### PR TITLE
Add to config to expose enableServiceLinks in pod spec

### PR DESCRIPTION
**Pull request by idlewis**
RCO portion of issue
OpenLiberty/open-liberty-operator#487

### CODE REVIEW

File: bundle/manifests/rc.app.stacks_runtimecomponents.yaml [hunk 1]
```diff
@@ -1033,6 +1033,8 @@ spec:
                         type: string
                     type: object
                 type: object
+              disableServiceLinks:
```
Comment (leochr) [inline bundle/manifests/rc.app.stacks_runtimecomponents.yaml:1036]: A description should be added:
Disable information about services being injected into the application pod's environment variables. Default to false.

**Review by leochr — COMMENTED** [review]
@idlewis Looks good. Just one comment about adding a description. Thanks

**Comment by idlewis** [comment]
@leochr I've added the description as requested. This is now ready for review.

**Review by leochr — APPROVED** [review]
@idlewis Looks good. Thank you

**Pull request merged**